core/arch/arm/kernel/thread.c

```diff
@@ -929,6 +929,49 @@ static void init_sec_mon(size_t pos __maybe_unused)
 #endif
 }
 
+static uint32_t __maybe_unused get_midr_implementer(uint32_t midr)
+{
+	return (midr >> MIDR_IMPLEMENTER_SHIFT) &
+	       (BIT(MIDR_IMPLEMENTER_WIDTH) - 1);
```

Comment:
It is a bit weird to see how you call `BIT()` macro with `_WIDTH` parameter. Maybe it is better to add proper definition for `MIDR_IMPLEMENTER_MASK` in header file?

Reply:
Yes, I'll do that.